versions.tf

```diff
@@ -1,8 +1,8 @@
 terraform {
   required_providers {
-    cloudscale = {
-      source  = "terraform-providers/cloudscale"
-      version = ">= 2.2"
+     cloudscale = {
```

Comment:
that whitespace required or harmful?

Reply:
No, not required and not harmfull. I have run `terraform fmt` to fix some formating issues like this.